src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,57 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.ByteArrayOutputStream
+
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  "printHelp" should "produce help info with values from custom properties file" in {
```

Comment:
This test does not go far enough: it should read the properties file and assert all the values appear as defaults. The keys of the test version should als match the keys in the assembly version of the properties files. Thus you assert everything gets properly maintained in the future.